src/app/shop/[category]/[productId]/_components/TabContents/Review/ReviewImageList.tsx

```diff
@@ -1,23 +1,83 @@
+'use client';
+
+import { Modal } from '@/components';
 import RenderImages from '@/components/ReviewItem/RenderImages';
+import { ChevronIcon } from '@/public/index';
 import type { ReviewImage } from '@/types/ProductReviewTypes';
 import classNames from 'classnames/bind';
+import { useState } from 'react';
 import styles from './ReviewImageList.module.scss';
+import ReviewImageModal from './ReviewImageModal';
 
 const cn = classNames.bind(styles);
 interface ReviewImageListProps {
+  productId: number;
   reviewImgs: ReviewImage[];
 }
 
-export default function ReviewImageList({ reviewImgs }: ReviewImageListProps) {
+const SECTION_SIZE = 7;
+
+export default function ReviewImageList({ productId, reviewImgs }: ReviewImageListProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [reviewId, setReviewId] = useState<number>(0);
+  const [currentSectionIndex, setCurrentSectionIndex] = useState(0);
+
+  const isPrevDisabled = currentSectionIndex === 0;
+  const isNextDisabled = currentSectionIndex === Math.floor(reviewImgs.length / SECTION_SIZE) - 1;
+
+  const handleModalOpen = (id: number) => {
+    setIsOpen(true);
+    setReviewId(id);
+  };
+
+  const handleModalClose = () => {
+    setIsOpen(false);
+  };
+
+  const handleMovePrevSection = () => {
+    if (currentSectionIndex > 0) setCurrentSectionIndex((prev) => prev - 1);
+  };
+  const handleMoveNextSection = () => {
+    if (currentSectionIndex < Math.floor(reviewImgs.length / SECTION_SIZE)) setCurrentSectionIndex((prev) => prev + 1);
```

Comment:
isPrevDisabled하고 isNextDisabled를 등호가 아닌 부등호로 하면 다음과 같이 적용할 수 있을 것 같아요!
```suggestion
  //const isPrevDisabled = currentSectionIndex <= 0;
  //const isNextDisabled = currentSectionIndex >= Math.floor(reviewImgs.length / SECTION_SIZE) ;
    
  const handleMovePrevSection = () => {
    if (!isPrevDisabled) {
      setCurrentSectionIndex((prev) => prev - 1);
    }
  };
  const handleMoveNextSection = () => {
    if (!isNextDisabled) {
      setCurrentSectionIndex((prev) => prev + 1);
    }
```